book_source/02_demos_tutorials_workflows/01_install_pecan.Rmd

```diff
@@ -80,7 +80,7 @@ This will not go into much detail about about how to use Docker -- for more deta
 
     This should print the current version of docker-compose. We have tested the instruction below with versions of docker-compose 1.22 and above.
    
-3. **Download the PEcAn docker-compose file**. It is located in the root directory of the [PEcAn source code](https://github.com/pecanproject/pecan). For reference, here are direct links to the [latest stable version](https://raw.githubusercontent.com/PecanProject/pecan/master/docker-compose.yml) and the [bleeding edge development version](https://raw.githubusercontent.com/PecanProject/pecan/master/docker-compose.yml). (To download the files, you should be able to right click the link and select "Save link as".) Make sure the file is saved as `docker-compose.yml`.
+3. **Download the PEcAn docker-compose file**. It is located in the root directory of the [PEcAn source code](https://github.com/pecanproject/pecan). For reference, here are direct links to the [latest stable version](https://raw.githubusercontent.com/PecanProject/pecan/master/docker-compose.yml) and the [bleeding edge development version](https://raw.githubusercontent.com/PecanProject/pecan/master/docker-compose.yml). (To download the files, you should be able to right click the link and select "Save link as".) Make sure the file is saved as `docker-compose.yml` in a directory called `pecan`.
```

Comment:
the other option is to use `docker-compose -p pecan ...`

Reply:
So my thinking here was that, if you take this approach then you have to remember to use `-p pecan` for _every_ `docker-compose` command, which is a bit of a pain. I'll also note that the `-p` flag is described in detail in the main Docker reference section, which these sections link to.